### ISSUE TITLE
What does "host" do

### ISSUE THREAD

**Issue by laoergege** [giscus/giscus-component#1003]
https://github.com/giscus/giscus-component/blob/2f941e1af225963e9a040653c71839a96333afc1/vue/src/lib/Giscus.vue#L43

What does "host" do, I've found that without it the iframe height doesn't adapt?

**Comment by laymonage** [comment]
> What does "host" do 

It can be used to use a different giscus server, e.g. if you run a self-hosted instance.

> I've found that without it the iframe height doesn't adapt?

What version of the component are you using? That issue should've been fixed recently.

**Comment by laoergege** [comment]
I use the v2.2.6 version of @giscus/vue.

I must add `host="https://giscus.app"` to fix the problem of iframe height.

```vue
<template>
  <Giscus
    id="comments"
    repo="laoergege/laoergege-blog"
    repoId="MDEwOlJlcG9zaXRvcnkxMjM4OTQ4MDE="
    categoryId="DIC_kwDOB2J8Ec4CTiA5"
    mapping="specific"
    :term="discussionID"
    strict="0"
    reactions-enabled="1"
    emit-metadata="0"
    input-position="top"
    :theme="compTheme"
    lang="zh-CN"
    loading="lazy"
    host="https://giscus.app"
  />
</template>
```

**Comment by laoergege** [comment]
If there is no value of "host", sometimes it may be rendered like the below.

```
<giscus-widget id="comments" repoid="MDEwOlJlcG9zaXRvcnkxMjM4OTQ4MDE=" categoryid="DIC_kwDOB2J8Ec4CTiA5" reactionsenabled="1" emitmetadata="0" inputposition="top" host="" strict="0" theme="dark_dimmed" lang="zh-CN" loading="lazy" repo="laoergege/laoergege-blog" mapping="specific" term="REvhMsxERv4HH56vL_O5B"></giscus-widget>
```

**Comment by youyinnn** [comment]
Yeah, this confuses me a lot. But the `host` do fixed the height issue.

**Comment by laymonage** [comment]
Finally found the cause, apparently Vue sets the `host` to an empty string if you don't specify it.

Should be fixed in v2.3.0, please update and confirm. Thanks!
